api/tests/unit/domain/models/profile_test.js

```diff
@@ -439,6 +446,118 @@ describe('Unit | Domain | Models | Profile', () => {
         expect(profile.areas).to.equal(areas);
 
       });
+    });
+
+    context('days before being able to replay a competence', () => {
+
+      let clock;
+      let now;
+
+      beforeEach(() => {
+        now = new Date('2018-01-10 05:00:00');
+        clock = sinon.useFakeTimers(now.getTime());
+      });
+
+      afterEach(() => {
+        clock.restore();
+      });
+
+      context('should be undefined', () => {
```

Comment:
je ne comprend pas la gestion des context ici. Un 'context' ne devrait pas ajouter de l'info quant au contexte du test ? et le 'it' tester quelque chose ('should ...') ?

Reply:
Ouai en fait j'ai fait mon TDD en fonction des résultats attendus, du coup ça match pas super bien avec les contexts
Je peux essayé de voir pour les réorganiser (si c'est pas trop relou)